transport_test.go

```diff
@@ -175,13 +176,20 @@ func gatherAndExchangeCandidates(aAgent, bAgent *Agent) {
 
 	candidates, err := aAgent.GetLocalCandidates()
 	check(err)
+
 	for _, c := range candidates {
+		if addr, err := netip.ParseAddr(c.Address()); err == nil {
+			if shouldFilterLocationTrackedIP(addr) {
```

Comment:
This is a nice check to make sure we never ever publish what are currently defined as location tracked IPs